countItems.js

```diff
@@ -0,0 +1,33 @@
+const { MongoClientInterface } =
+    require('arsenal').storage.metadata.mongoclient;
+const { Logger } = require('werelogs');
+
+const log = new Logger('S3Utils::ScanItemCount');
+const replicaSetHosts = process.env.MONGODB_REPLICASET;
+
+const config = {
+    replicaSetHosts,
+    writeConcern: 'majority',
+    replicaSet: 'rs0',
+    readPreference: 'primary',
+    database: 'metadata',
+    replicationGroupId: 'RG001',
+    logger: log,
+};
+
+const mongoclient = new MongoClientInterface(config);
+mongoclient.setup(err => {
+    if (err) {
+        return log.error('error connecting to mongodb', err);
+    }
+    return mongoclient.scanItemCount(log, error => {
+        if (error) {
+            log.error('error occurred during mongo scan item count', {
+                error,
+            });
+            return process.exit(1);
+        }
+        log.info('mongo scan item count has completed');
+        return process.exit(0);
+    });
```

Comment:
remarks: not sure how mongodb handles dead connections, so manually closing the connection maybe a good idea?